provisioning/local/gcp_kms.tf

```diff
@@ -0,0 +1,23 @@
+resource "google_kms_key_ring" "object_encryptor_keyring" {
+  name     = "${var.name_prefix}-object-encryptor"
+  location = "global"
+}
+
+resource "google_kms_crypto_key" "object_encryptor_key" {
+  name     = "${var.name_prefix}-object-encryptor"
+  key_ring = google_kms_key_ring.object_encryptor_keyring.id
+  purpose  = "ENCRYPT_DECRYPT"
+
+  lifecycle {
+    prevent_destroy = false
+  }
```

Comment:
dtto, co se tu stane na pozadi, kdyz udelam destroy a znovu apply?

Reply:
nejspíš se to vyjebe, protože ten klíč má soft delete a nejde smazat ani přes konzoli, stackoverflow workaround je založ si to s novým namem 🤷‍♂️  (btw na azure je to vlastně stejný, akorát tam zrovna nepoužíváme keys, ale secrets)